components/server/src/user/user-service.ts

```diff
@@ -148,6 +148,14 @@ export class UserService {
         if (token) {
             await this.userDb.storeSingleToken(identity, token);
         }
+
+        if (
```

Comment:
I'd have expected that new users are by default on the "migrated" path without needing to run the migration. The consequence is that we'll need to end up running this code even after we've migrated all of them (or we pull in the logic here by default).

This is not a strict problem, but feels odd. Do you think there's a way to have the new user default not require the migration?

Reply:
For new users, the migration really just creates an org, because there is nothing else that needs to be migrated. The migration logic would also trigger if users are not a member of any org (i.e. they have deleted the last org). Which I consider the same case as with a new user that doesn't have an org.
Maybe it would make sense to not automatically create an org, but send users without an org (new users or not) to the create org flow? cc @atduarte 
In any case, for this PR, I prefer to call the same logic instead of having a second place where we create an artificial org.

Reply:
For now I would prefer to go with the simplest solution that doesn't introduce further migration challenges down the line. 

We are now figuring out Dedicated onboarding—which is org-first, user-second—and that might lead to changes in gitpod.io. _Personally think onboarding in gitpod.io should also be org-first, user-second._ Therefore avoiding extra work that might be wasted sounds best.